README.md

```diff
@@ -68,7 +68,7 @@ To ensure the long-term quality of the GX Core codebase, we're not yet ready to
 
 | GX Component         | Readiness          | Notes |
 | -------------------- | ------------------ | ----- |
-| Action               | 🟢 Ready           |       |
+| Action               | 🔴 Not ready       |       |
```

Comment:
Do we want to move this down? Looks like we sort by Readiness?

Reply:
ah, yes, good catch!